OOP/assignment2/SurveyApplication.java

```diff
@@ -0,0 +1,204 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class SurveyApplication {
+	private static ArrayList<Question> questionList = new ArrayList<Question>();
+	private static ArrayList<Particpant> participantList = new ArrayList<Particpant>();
+
+	public static void main(String args[]) {
+		questionList.add(new Question("Text", "Give your brief feedback"));
```

Comment:
ACTION: All the questions should be read from a text file as given in the requirement. Read the problem statement once again.